external-crates/move/crates/move-compiler/tests/move_2024/hlir/dead_code_block_match_return_valid.snap

```diff
@@ -0,0 +1,19 @@
+---
+source: crates/move-compiler/tests/move_check_testsuite.rs
+info:
+  flavor: core
+  edition: 2024.alpha
+  lint: false
+---
+warning[W09005]: dead or unreachable code
+  ┌─ tests/move_2024/hlir/dead_code_block_match_return_valid.move:5:21
+  │  
+5 │           match (0u8) {
+  │ ╭─────────────────────^
+6 │ │             255 => return 'a true,
+7 │ │             0 => return 'a false,
+8 │ │             _ => return 'a false,
```

Comment:
Whats going on with these `,`s?

Reply:
I think this is just a weird formatting thing with `.snap`